mlrun/api/crud/model_monitoring/_model_endpoint_stores/__init__.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import enum
+import typing
+
+import mlrun
+
+from .model_endpoint_store import ModelEndpointStore
+
+
+class ModelEndpointStoreType(enum.Enum):
+    """Enum class to handle the different store type values for saving a model endpoint record."""
+
+    kv = "kv"
+    sql = "sql"
+
+    def to_endpoint_target(
+        self,
+        project: str,
+        access_key: str = None,
+        connection_string: str = None,
+    ) -> ModelEndpointStore:
+        """
+        Return a ModelEndpointStore object based on the provided enum value.
+
+        :param project:           The name of the project.
+        :param access_key:        Access key with permission to the DB table. Note that if access key is None and the
+                                  endpoint target is from type KV then the access key will be retrieved from the
+                                  environment variable.
+        :param connection_string: A valid connection string for SQL target. Contains several key-value pairs that
+                                  required for the database connection.
+                                  e.g. A root user with password 1234, tries to connect a schema called mlrun within a
+                                  local MySQL DB instance: 'mysql+pymysql://root:1234@localhost:3306/mlrun'.
+
+        :return: ModelEndpointStore object.
+
+        """
+
+        if self.value == ModelEndpointStoreType.kv.value:
```

Comment:
No it's not, because in that case you are basically comparing type `str` with type `enum 'ModelEndpointStoreType'` so the result will always be `False` (I've tried it...). Therefore, we should keep it as is - `ModelEndpointStoreType.kv.value`.